pkg/webui/lib/shared-messages.js

```diff
@@ -215,6 +215,7 @@ export default defineMessages({
   validateTooLong: 'Too long',
   validateTooShort: 'Too short',
   validateUrl: 'Invalid URL format',
+  validateUserFormat: 'Username can contain only dash (-) as the special char',
```

Comment:
(any) ID can be 3-36 characters (already covered by your other validations) and can only contain lowercase letters, numbers and dashes (-)

Reply:
alright. there are some cases not covered though, but in my opinion its too specific to be mentioned. these cases are if the username is for example `p-g` or any id that has dash in the end `tti-` for example.

Reply:
I agree. If someone tries to use `----` as a username, they can probably already imagine that we wouldn't accept it.